app/views/admin/people/edit.html.erb

```diff
@@ -0,0 +1,4 @@
+<center><h1 class="droidFont">Editing Person</h1></center>
```

Comment:
Mostly a great pull request, @hyanying .  One thing I see is this use of class"droidFont" in places like the above.  This feels a bit strange to me.  If you want the H1s to use that font, why not add a style for H1s to accomplish this?